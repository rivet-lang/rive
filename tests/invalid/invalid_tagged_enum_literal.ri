enum TaggedEnum {
    Valid,
    Invalid(int)
}

func main() {
    _ = TaggedEnum.Valid;
    _ = TaggedEnum.Invalid;
}
