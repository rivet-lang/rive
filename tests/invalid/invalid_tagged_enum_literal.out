tests/invalid/invalid_tagged_enum_literal.ri:8:9: error: variant `Invalid` cannot be initialized without arguments
    8 |     _ = TaggedEnum.Invalid;
      |         ^~~~~~~~~~~~~~~~~~
      = help: if you intend not to pass any value, add `()`: `Invalid()`
rivet: error: could not compile module `invalid_tagged_enum_literal`, aborting due to previous error